### DISCUSSION TITLE
In the future, will the API stay free for reasonable limits?

### DISCUSSION THREAD

**Discussion by Secret-chest** [StractOrg/stract#188] [Q&A] [answered]
If I want to build a custom frontend for _humans_ to use, is that OK?

**Comment by mikkeldenker** [comment, marked as answer]
Yes that's totally fine. I haven't decided on the exact pricing for the API yet, but it will probably be ~$5 for 1000 searches with some amount of free searches each month. It will probably be around 1000 free searches/month, but I'm not exactly sure yet

**Reply by Secret-chest** [reply]
Thanks for staying open to things like these!
